### ISSUE TITLE
By-default picture parameters

### ISSUE THREAD

**Issue by overflorian** [nobelization/panoramax-mobile-app#40]
I suggest the default settings while opening the camera is defined as:
* 16/9 ratio
* infinite focus
* iso/exposure is forced on 1/2 bottom of the picture, above the red line:

![image](https://github.com/nobelization/panoramax-mobile-app/assets/11553211/b606e219-a7ef-4077-8c1d-7047c9487cd9)

This would avoid the picture exposure to focus on the sky https://panoramax.openstreetmap.fr/#background=streets&focus=pic&map=16.94/49.193114/-0.383608&pic=71386ed8-c78a-4f4e-a3b8-9e4d41ce506d&speed=250&xyz=109.72/-7.14/30

**Comment by mnalis** [comment]
> iso/exposure is forced on 1/2 bottom of the picture, above the red line:

You mean, **below** the red line?

**Comment by overflorian** [comment]
Yes, because most of the time, the sky is above and we would like to ignore the sky luminosity.

**Comment by chrisdebian** [comment]
What are the current default settings? over Florian's suggestion seems eminently sensible.

Chris
